### DISCUSSION TITLE
Invalid search plugin due to namespace not matching.

### DISCUSSION THREAD

**Discussion by jdepumpo** [searxng/searxng#2207] [Q&A]
I'm having an issue with the OpenSearch xml and adding it to Firefox. I get "Invalid Format: Firefox could not install the search plugin."
When I inspect the XML I see: "Invalid search plugin due to namespace not matching." In terms of URLs the XML looks like it is generated correctly.

Any ideas? Thanks.

**Comment by return42** [comment]
Is the SearXNG instance public?

**Reply by jdepumpo** [reply]
No, it is behind Cloudflare Access.

**Reply by return42** [reply]
Sorry we can't help with Cloudflare setups.
